addons/kotlin/build_dialog/BuildDialog.gd

```diff
@@ -0,0 +1,41 @@
+######################################
```

Comment:
I really think this button should be closable and then the build progress is shown somwhere else. Ex beside the Build Kotlin button as a spinner or something like that. now the user is blocked during build and cannot tinker on a scene for example

Reply:
This is a good idea, however it'll be a bit of a larger change. I think we should log this as an improvement.

Reply:
Agreed I'll make an issue later